manifest/run.go

```diff
@@ -238,23 +239,22 @@ func pruneSyncs(syncs []sync.Sync) []sync.Sync {
 	return pruned
 }
 
-func waitForContainer(container string, service Service) error {
-	i := 0
+func waitForContainer(container string, service Service, wait int) error {
+	waitTimeout := time.add(time.parseDuration("%ds", wait))
```

Comment:
I rewrote this to use actual time. At least I am hoping I did. The idea was to say, "now+configured interval".

(Disclaimer: I think I've never written that much Go before.)